Tests/BootCamp.Chapter.Tests/CardTests.cs

```diff
@@ -1,32 +1,22 @@
 ﻿using BootCamp.Chapter.Gambling;
+using System.Linq.Expressions;
 using Xunit;
 using static BootCamp.Chapter.Gambling.Card;
 
 namespace BootCamp.Chapter.Tests
 {
     public class CardTests
     {
-        private readonly Card card;
-        private readonly Suites expectedSuite = Suites.Diamonds;
-        private readonly Ranks expectedRank = Ranks.Eight;
-
-        public CardTests()
-        {
-            card = new Card(expectedSuite, expectedRank);
-        }
-
-        [Fact]
-        public void Card_When_Given_Suite_Sets_Suite() => Assert.Equal(expectedSuite, card.Suite);
-
-        [Fact]
-        public void Card_When_Given_Suite_Sets_Ranks() => Assert.Equal(expectedRank, card.Rank);
-
         [Fact]
         public void Card_When_Equivalent_Cards_Returns_True()
```

Comment:
Card will never be equivalent to cards. Please more careful with the naming. should be Card_When_Equivalent_**Card**_Returns_True